src/toolkit/components/pdfjs/content/build/pdf-mjs.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/toolkit/components/pdfjs/content/build/pdf.mjs b/toolkit/components/pdfjs/content/build/pdf.mjs
+index 9798d4f8f9473eac434ff1a521b3728bea436723..144d42b2830a7d9eb345db18a018cd8741753c3c 100644
+--- a/toolkit/components/pdfjs/content/build/pdf.mjs
++++ b/toolkit/components/pdfjs/content/build/pdf.mjs
+@@ -22,7 +22,7 @@
+ 
+ /******/ // The require scope
+ /******/ var __webpack_require__ = {};
+-/******/ 
++/******/
```

Comment:
Was this intentional?

Reply:
Oh, I meant to remove this but I guess it got added again after force pusing